src/assets/images/icons/Flow/Export.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20px" height="18px" viewBox="0 0 20 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Icon/Duplicate Copy 7</title>
```

Comment:
Check out the below image, please update the title to something meaningful.
![image](https://user-images.githubusercontent.com/26791171/123627188-53726300-d82f-11eb-8a6e-b592e1a24ace.png)